README.md

```diff
@@ -20,7 +20,7 @@ https://codesoapbox.dev/keycloak-user-migration
 
 | Keycloak Version | Commit                                                                                                                                           |
 |------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
-| 11.X             | Current                                                                                                                                          |
+| 12.X             | Current                                                                                                                                          |
```

Comment:
We should keep 11.X (alongside 12.X being `Current`) as pointing to the last commit with Keycloak 11 dependencies.

Reply:
Ah, good point! Added a link to the last commit now.